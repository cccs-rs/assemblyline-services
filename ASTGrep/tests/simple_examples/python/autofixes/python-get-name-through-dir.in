getattr(xxx,dir(__import__('builtins'))[dir(__import__('builtins')).index('bool')])

getattr(__import__('builtins'),dir(__import__('builtins'))[dir(__import__('builtins')).index(aaa)])
