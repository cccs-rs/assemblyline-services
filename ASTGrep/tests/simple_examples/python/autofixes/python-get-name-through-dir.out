getattr(xxx,'bool')

getattr(__import__('builtins'),dir(__import__('builtins'))[dir(__import__('builtins')).index(aaa)])
